### PR TITLE
Add fullstop to end of sentence

### PR DESCRIPTION
**Pull request by HonkingGoose**
### Why:

Fixing a grammar problem

### What's being changed (if available, include any code snippets, screenshots, or gifs):

- Add full stop to end of the sentence

The rest of this document adds the `.` inside the `"` character when linking to another GitHub Docs page, so I'm following that convention in my PR.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ on:
   merge_group:
 ```
 
-For more information see "[Events that trigger workflows](/actions/using-workflows/events-that-trigger-workflows#merge-group)"
+For more information see "[Events that trigger workflows](/actions/using-workflows/events-that-trigger-workflows#merge-group)."
```
Comment (cmwilson21) [inline content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue.md:51]:
```suggestion
For more information, see "[Events that trigger workflows](/actions/using-workflows/events-that-trigger-workflows#merge-group)."
```

**Review by cmwilson21 — CHANGES_REQUESTED** [review]
Great catch with this one. I noticed it's missing a comma as well, so I'm going to go ahead and add it here while we are making this change ✨

**Review by cmwilson21 — DISMISSED** [review]

**Review by cmwilson21 — APPROVED** [review]
LGTM! Thank you for this! 🚢

**Comment by HonkingGoose** [comment]
Thank you for reviewing and merging my change! ❤️

**Pull request merged**